_includes/head-custom.html

```diff
@@ -0,0 +1,9 @@
+<!-- start custom head snippets -->
```

Comment:
Thinking a bit harder about these helpful comments. Should we include the filename in this comment, maybe? I'm just thinking about how a user would figure out how to customize these. What do you think? Would you find this helpful if you were using this theme?

```suggestion
<!-- start custom head snippets, customize with your own _includes/head-custom.html file -->
```

Reply:
Indeed, I think that would be helpful. Maybe in addition, we should add something to the `README.md` to mention in the `Layouts` section that instead of completely replacing the layout, some changed can be added by adding custom includes, based on the files provided with the theme? 
I'll add something along these lines in an additional commit so you can have a look :-).